kratos/mpi/tests/test_utilities/mpi_cpp_test_utilities.h

```diff
@@ -47,6 +47,18 @@ static void GenerateDistributedTriangleMesh(
     const DataCommunicator& rDataCommunicator
     );
 
+/**
+* @brief It generates a Triforce mesh
+* @details  ▲
+*          ▲ ▲
+* @param rModelPart The model part to be filled
+* @param rDataCommunicator The data communicator
+*/
+static void GenerateDistributedTriforceMesh(
```

Comment:
🤦‍♂️ la madre que te parió...

Reply:
Casualmente tenia un problema que fallaba con una distribucion similar. Lo podria haber llamado de otra manera, como ClanHojo test: https://es.wikipedia.org/wiki/Clan_H%C5%8Dj%C5%8D, es el origen de la trifuerza de acuerdo con Miyamoto san.

![actually-nerd](https://github.com/user-attachments/assets/c425f7ee-5602-43d3-a28d-4d0ecf77da85)

Reply:
https://github.com/KratosMultiphysics/Kratos/pull/12726#discussion_r1787674758

:stuck_out_tongue:

Reply:
I didn't start in Spanish...

Reply:
> I didn't start in Spanish...

I didnt mean you :D